scripts/protocol/mina/get_mina_state.py

```diff
@@ -63,16 +64,58 @@ def state_query_to_graphql(output_path, url):
     print("Hash: {}".format(protocol_state["protocolState"]["blockchainState"]["snarkedLedgerHash"]))
     return request_res
 
+
+def get_mina_account_state(url, address):
+    query = '''
+    query {{
+      account(publicKey: "{0}" ) {{
+        index
+        zkappState
+        balance {{
+          liquid
+          locked
+          stateHash
+        }}    
+        leafHash
+        receiptChainHash
+        merklePath {{
+          left,
+          right
+        }}
+      }}
+    }}
+    '''.format(address)
+    request_res = requests.post(url, json={"query": query}).json()
+    return request_res
+
+
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
-                    prog = 'Mina Helper for =nil; Proof Market',
-                    description = 'Mina Helper retrieves data related to Proof Market functionality from Mina node')
-    parser.add_argument('--url', help="GraphQL URL", default="http://localhost:3085/graphql")
-    parser.add_argument('--output', help="Output file path", default="proof_market_data.json")
+        prog='Mina Helper for =nil; Proof Market',
+        description='Mina Helper retrieves data related to Proof Market functionality from Mina node')
+    parser.add_argument('--url', help="GraphQL URL", default="https://proxy.berkeley.minaexplorer.com/")
+    parser.add_argument('--output', help="Output file path", default="output.json")
+    parser.add_argument('--address', help="Mina public key of zkApp or user", default="")
+    parser.add_argument('--type', help="Query type: ledger or account", default="")
```

Comment:
I think, it's better to do it via subparsers. 
Example:
https://github.com/NilFoundation/proof-market-toolchain/blob/master/scripts/ask_tools.py#L56

Reply:
Updated